c++/src/kj/async-io-test.c++

```diff
@@ -409,6 +452,10 @@ TEST(AsyncIo, AddressParsing) {
   EXPECT_EQ("unix-abstract:foo/bar/baz", tryParse(w, network, "unix-abstract:foo/bar/baz"));
 #endif
 
+#if __linux__
+  EXPECT_EQ("vsock:4294967295:1234", tryParse(w, network, "vsock:-1:1234"));
```

Comment:
Is this coercion of -1 to an unsigned value common expected behavior when dealing with vsocks? Otherwise to my unfamiliar eye it looks like a bug.